daemon/src/oracle.rs

```diff
@@ -56,11 +55,11 @@ struct NewAnnouncementFetched {
     nonce_pks: Vec<schnorrsig::PublicKey>,
 }
 
-/// A module-private message to allow parallelization of fetching attestations.
-#[derive(Debug)]
-struct NewAttestationFetched {
+/// A message to allow parallelization of fetching attestations.
+#[derive(Debug, Clone)]
+pub struct NewAttestationFetched {
```

Comment:
Why is this suddenly public?

Reply:
Its used in the tests so I had to make it pub

Reply:
Yes, you added the handler the test but that one is not actually needed. I just tried it locally and it can be deleted without problems! :)

We only send this message to ourselves so it doesn't really make sense that anyone else needs to know about this message!